src/datasources/__tests__/mapquest.js

```diff
@@ -0,0 +1,32 @@
+const mockData = require("./mock_data/mockMapquestResponses");
+const MapQuestAPI = require("../mapquest");
+
+const mocks = {
+  get: jest.fn()
+};
+
+const datasource = new MapQuestAPI();
+datasource.get = mocks.get;
+
+//Reducers
+describe("[MapQuestAPI Reducers]", () => {
+  describe("postalDataReducer", () => {
+    it("properly transforms postal data", () => {
+      expect(datasource.postalDataReducer(mockData.mockLocation)).toEqual(
+        mockData.mockPostalDataReduced
+      );
+    });
+  });
+});
+
+//Queries
+describe("[MapQuestAPI Queries]", () => {
```

Comment:
This is a redundant test because you are processing the exact same kind of object in the same way as the test above it. However, it is probably good to have it separated out like this in case we want to modify either in the future. Just keep that in mind as a potential refactor point for less lines of code.